recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "pysolar" %}
-{% set version = "0.6" %}
+{% set version = "0.8rc1" %}
```

Comment:
The bot was probably mislead by the fact that the version in URL version was hard-coded.
Still, the bot should not be issuing PRs for RC.

@CJ-Wright I guess that this PR is from before this was fixed in the bot, right?

Reply:
Most likely